lib/Doctrine/ORM/Configuration.php

```diff
@@ -79,45 +73,35 @@ public function getAutoGenerateProxyClasses()
      * @param bool|int $autoGenerate Possible values are constants of Doctrine\Common\Proxy\AbstractProxyFactory.
      * @psalm-param bool|AutogenerateMode $autoGenerate
      * True is converted to AUTOGENERATE_ALWAYS, false to AUTOGENERATE_NEVER.
-     *
-     * @return void
      */
-    public function setAutoGenerateProxyClasses($autoGenerate)
+    public function setAutoGenerateProxyClasses(bool|int $autoGenerate): void
     {
         $this->_attributes['autoGenerateProxyClasses'] = (int) $autoGenerate;
     }
 
     /**
      * Gets the namespace where proxy classes reside.
-     *
-     * @return string|null
      */
-    public function getProxyNamespace()
+    public function getProxyNamespace(): string|null
     {
         return $this->_attributes['proxyNamespace'] ?? null;
     }
 
     /**
      * Sets the namespace where proxy classes reside.
-     *
-     * @param string $ns
-     *
-     * @return void
      */
-    public function setProxyNamespace($ns)
+    public function setProxyNamespace(string $ns): void
     {
         $this->_attributes['proxyNamespace'] = $ns;
     }
 
     /**
      * Sets the cache driver implementation that is used for metadata caching.
      *
-     * @return void
-     *
      * @todo Force parameter to be a Closure to ensure lazy evaluation
      *       (as soon as a metadata cache is in effect, the driver never needs to initialize).
```

Comment:
I don't understand this todo

Reply:
I think, the author wants that we can set a `Closure(): MappingDriver` instead of a `MappingDriver` instance here.

Reply:
Ok… I think there is a number of todos we could challenge in this project.